lib/bamboozled/api/base.rb

```diff
@@ -35,7 +35,11 @@ def request(method, path, options = {})
           case response.code
           when 200..201
             begin
-              JSON.parse(response).with_indifferent_access
+              if response.nil?
+                {"headers" => response.headers}.with_indifferent_access
```

Comment:
hey @Enriikke when its a successful POST request `response.body` is an empty string and `response.headers` is something like this: `{"date"=>["Thu, 23 Jul 2015 14:19:11 GMT"], "server"=>["Apache"], "location"=>["https://api.bamboohr.com/api/gateway.php/alphasights/v1/employees/44902"], "vary"=>["User-Agent"], "content-length"=>["0"], "connection"=>["close"], "content-type"=>["text/html; charset=UTF-8"]}` so it actaully shouldn't blow up and doing the check with `response.body.nil` wouldn't work.

I would prefer to do `response.body.empty?` however was having trouble getting the .yml fixture to return an empty string for `response.body` for testing purposes. I could only get it to return nil or something like `" '' "` which isn't correct either.  Any suggestions on how to do this would be much appreciated.